ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml

```diff
@@ -1667,6 +1667,7 @@ tests:
       AUX_HOST: openshift-qe-bastion.arm.eng.rdu2.redhat.com
       E2E_RUN_TAGS: '@baremetal-upi and @arm64 and @network-ovnkubernetes and not
         @fips'
+      FIPS_ENABLED: "true"
```

Comment:
arm64 does not support FIPS (due to the missing support in vendor HW and in RHEL). This PR should not consider ARM64.

Reply:
Thanks, we should remove then.

Reply:
> Thanks, we should remove then.

Removed fips in arm platform.